### DISCUSSION TITLE
Keybind - prayer actions

### DISCUSSION THREAD

**Discussion by DaGoat15** [runelite/runelite#16874] [Ideas]
Since RS3 allows keybinds for overhead prayers and the osrs team has openly said 1 to 1 actions are ok to use. Could there be a plugin that would allow this for pvm? I could see how this could be an issue for pvp for people having a unfair advantage. But for pvm this would be such a huge QOL update. Maybe even figure out a way so the plugin turns off in pvp areas.

**Comment by Nightfirecat** [comment]
Keybinds are typically not analagous to clicks in specific locations, so this isn't a 1-to-1 mapping. At this time, we're not looking to add this feature.

**Reply by DaGoat15** [reply]
Good to know. I kinda wish they did more with the game for small stuff like keybinds. I just know that from playing rs3 prayer flicking or changing prayer overheads are so much easier with keybinds.
